ui/src/main/java/my/painboard/service/config/WebConf.java

```diff
@@ -1,26 +1,36 @@
 package my.painboard.service.config;
 
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
 import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
 
 @EnableWebMvc
 @Configuration
-@ComponentScan(basePackages = {"my.painboard.service.controller", "my.painboard.service.filesystem"})
-public class WebConf extends WebMvcConfigurerAdapter {
-    private static final String[] CLASSPATH_RESOURCE_LOCATIONS = {"classpath:/static/"};
-
-    @Override
-    public void addResourceHandlers(ResourceHandlerRegistry registry) {
-        registry.addResourceHandler("/**").addResourceLocations(CLASSPATH_RESOURCE_LOCATIONS);
-    }
+@ComponentScan(basePackages = {"my.painboard.service.controller"})
+public class WebConf implements WebMvcConfigurer {
+//    private static final String[] CLASSPATH_RESOURCE_LOCATIONS = {"classpath:/static/"};
+//
+//    @Override
+//    public void addResourceHandlers(ResourceHandlerRegistry registry) {
+//        registry.addResourceHandler("/**").addResourceLocations(CLASSPATH_RESOURCE_LOCATIONS);
+//    }
 
+//    @Override
+//    public void addViewControllers(ViewControllerRegistry registry) {
+//        registry.addViewController("/").setViewName("/index.html");
+//    }
 
-    @Override
-    public void addViewControllers(ViewControllerRegistry registry) {
-        registry.addViewController("/").setViewName("/index.html");
+    @Bean
+    public InternalResourceViewResolver viewResolver() {
+        InternalResourceViewResolver resolver = new InternalResourceViewResolver();
+        resolver.setPrefix("classpath:/static/");
+        resolver.setPrefix(".html");
+        resolver.setViewNames("index");
+        return resolver;
```

Comment:
I can not open index.html with these changes - why WebMvcConfigurer didn't work for you ?

Reply:
Now it works, check please)))